src/_playground/documentation/Properties/Properties.js

```diff
@@ -28,7 +31,7 @@ export const Properties = ({ sourceModule }) => {
 };
 
 Properties.propTypes = {
-    sourceModule: PropTypes.object.isRequired
+    sourceModule: PropTypes.string.isRequired
```

Comment:
Since this is being passed in as a string now, the prop name should probably change as well.  Similar to the new `getSourceModules` util function, maybe use `sourceModulePath`?